src/runtime/composables/useDirectusUser.ts

```diff
@@ -16,7 +16,7 @@ export const useDirectusUser = async () => {
     }
   })
 
-  const user = await directus.request(readMe()).catch((e) => {
+  const user = await directus.request(readMe()).catch((e: any) => {
```

Comment:
No explicit any, please.

Reply:
Is this a similar issue to the type cast? So that the actual error could define its type?